docs/releases.md

```diff
@@ -28,6 +28,7 @@ will be allowed up to release day.
 - [ ] publish dist.ipfs.io
 - [ ] publish next version to https://github.com/ipfs/npm-go-ipfs
 - [ ] publish gx version with `gx release`, as per [gx release guidelines](https://github.com/whyrusleeping/gx#publishing-and-releasing)
+- [ ] publish image to [Docker Hub](https://hub.docker.com/r/ipfs/go-ipfs/tags/) and update `latest` and `release` tags to point at it
```

Comment:
The publish should be automatic, but I think it takes a while. We should build image on one machine but then just update the tags. 

I think we can setup the automatic build in Docker Hub to be automatic and tag images based on branches... That way, we can have a `release` branch that would point to the `release` tag in Docker Hub, and you'll only need to update that and push it to update the Docker Hub tags.

Reply:
While we are at this: are we able to update or remove the `latest` tag?
IMO it is too ambiguous, and keeping `release` and `master` may be a better idea.
Alternatively it should be automated to point at the same things as `release`

Reply:
`latest` is de-facto (or maybe just is) standard in Docker-land, so we should probably keep it. Having `latest` being in sync with `release` I think is what we've agreed before and would map best to how things usually are dealt with. `master` would be the "nightly" basically with all the latest changes.